inc/Engine/Optimization/Minify/JS/Combine.php

```diff
@@ -202,6 +212,21 @@ function( $script ) {
 						return;
 					}
 
+					if (
```

Comment:
This part is being used twice, one inside the condition checking if url is external and one if not external.
so this code will be executed twice when url is external, right?

Reply:
It happens only once for each type: once for external, once for internal.

Reply:
If so, I think making it inside a new method would be better

Reply:
sorry I just saw the return keyword,fine.
it's minor change, I will approve PR and you can take the decision of adding a new local method for these two checks ot not